apps/src/bin/quiche-server.rs

```diff
@@ -187,129 +217,114 @@ fn main() {
             false => clients.values().filter_map(|c| c.conn.timeout()).min(),
         };
 
+        trace!("Polling the event loop for I/O events");
```

Comment:
this will be too spammy

Reply:
removed 👍 , along with another one that was also spammy.